src/data/DataStore.ts

```diff
@@ -983,6 +990,10 @@ class DataStore {
                 }
             }
 
+            if (countNaN > 0 && countNaN < frameEnd - frameStart) {
```

Comment:
I'm not sure what's this `countNaN < frameEnd - frameStart` check for?

Reply:
In a bucket, if all data is NaN, execute the original logic without separate processing. `countNaN < frameEnd - frameStart` can avoid adding multiple times.

```
// set frameSize = 3；
const dimStore = [11, 22, 20, 33, 44, 55]; 
// IF result index is [1, 5]
```

Demo 1:
```
const dimStore = [null, 22, 20, 33, 44, 55]; 
// result index is [0, 1, 5]
```

Demo 2:
```
const dimStore = [null, null, null, 33, 44, 55]; 
// result index is [0, 5]
// if not check, result will become [0, 0, 5].
```